src/kOS/Module/kOSProcessor.cs

```diff
@@ -30,6 +30,15 @@ public class kOSProcessor : PartModule, IProcessor, IPartCostModifier, IPartMass
 
         public Harddisk HardDisk { get; private set; }
 
+        public string Tag
+        {
+            get
+            {
+                KOSNameTag tag = part.Modules.OfType<KOSNameTag>().FirstOrDefault();
+                return tag != null ? tag.nameTag : null;
```

Comment:
No, I'm pretty sure that just gives the ability to tell if a variable has been defined.  There was a conscious decision at the time to not pass null values to the user, nor give the user an ability to access them.  Some way to null check will probably come when we implement a type structure for the users.